pyanaconda/network.py

```diff
@@ -34,6 +34,7 @@
 from uuid import uuid4
 import itertools
 import glob
+import logging
```

Comment:
It should be - it sets-up its own separate logger & log file for ficfg related logging, so I don't really see a need to change it.